ONSdigital/takeon-egestion
# Feature spp2891 new contract

# Description
contract changes

# How to test?
env: spp2891

To get 'new' contract change  
Set the DEFAULT_SCHEMA to 'false' in the export lambda
pass the following message into the 'spp-es-takeon-egestion-spp2891-db-export-input' queue  
check if the correct file has been generated in the 'spp-es-takeon-egestion-spp2891-export' bucket

# Links
https://collaborate2.ons.gov.uk/jira/browse/SPP-2891
